jax/util.py

```diff
@@ -16,6 +16,7 @@
   HashableFunction as HashableFunction,
   as_hashable_function as as_hashable_function,
   cache as cache,
+  get_logger as get_logger,
```

Comment:
This should not be a public API. It serves no additional value over logging.getLogger("jax"). Just revert this change?